preparingData.R

```diff
@@ -32,10 +33,9 @@ if (! require(readr, quietly = TRUE)) {
 # To read STRING data, it first needs to be downloaded from database 
 # and put into our /data directory. We've already done that for you
 # EXPLAIN RDS and saving space and loading RDS objects
-readr::read_delim("4932.protein.links.full.v11.0.txt", delim = " ")
-saveRDS(STR, "./data/4932.protein.links.full.v11.0")
 
-STR <- readRDS("./data/4932.protein.links.full.v11.0")
+onlineData = gzcon(url("https://string-db.org/download/protein.links.full.v11.0/4932.protein.links.full.v11.0.txt.gz"))
+STR <- read_delim(onlineData, delim = " ")
```

Comment:
yes, but let's leave in the readr:: syntax beacuse I feel like that comes up a lot / really helps in parsing others' code, especially if they're trolling StackOverflow for answers. I'll add a quick description of it to the file itself as well!